meta-linaro-mbl/meta-optee/recipes-security/optee/optee-os_%.bbappend

```diff
@@ -67,13 +67,14 @@ OPTEE_ARCH = "arm32"
 #
 EXTRA_OEMAKE = " \
                 CFG_DT=y \
-                CFG_TEE_CORE_LOG_LEVEL=1 \
                 LDFLAGS= \
                 LIBGCC_LOCATE_CFLAGS=--sysroot=${STAGING_DIR_HOST} \
                 NOWERROR=1 \
                 PLATFORM=${OPTEEMACHINE} \
          "
 
+EXTRA_OEMAKE += "${@bb.utils.contains("PACKAGECONFIG", "silent", " CFG_TEE_CORE_LOG_LEVEL=0", " CFG_TEE_CORE_LOG_LEVEL=1", d)}"
```

Comment:
Please, before this line adds the following otherwise bitbake will print a warning about invalid PACKAGECONFIG setting:
`PACKAGECONFIG[silent] = ""`

Reply:
fixed